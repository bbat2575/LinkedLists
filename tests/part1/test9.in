VIEW 0
NEW 4
1
2.2
E
Four
NEW 1
2e4
VIEW +1
VIEW -1
VIEW +0
VIEW -0
VIEW +
VIEW -
VIEW 2
VIEW o
VIEW O
VIEW ZERO
VIEW .
VIEW 0.
VIEW .0
VIEW 0.0
VIEW 0e0
VIEW 2e-6
VIEW 0 0
VIEW  0
VIEW   0
VIEW
VIEW  
  VIEW
0VIEW
0 VIEW
VIEW0
 VIEW 0
  VIEW 0
VIE W 0
VI EW 0
V IEW 0
view 0
View 0
VieW 0
VIeW 0
VEIW 0
VIEW  ALL
VIEW AL L
VIEW A LL
 VIEW ALL
  VIEW ALL
