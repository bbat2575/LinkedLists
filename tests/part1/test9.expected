
Type HELP for a list of options
Press <CTRL + D> to Quit

> INVALID COMMAND: VIEW

> List 0: 1 -> 2.20 -> E -> Four

> List 1: 20000.00

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: VIEW

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
